src/traces/scatter/attributes.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
             'See `y0` for more info.'
         ].join(' ')
     },
+    key: {
```

Comment:
`trace.uid` is a single key per trace. `key` is an array of keys for every data point in a trace. So if some data points drop out, object constancy is maintained, i.e. http://rickyreusser.com/animation-experiments/#error-bars-4 (if you can ignore the fill/line issue for a moment, the points are matched by key instead of by array index)